src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4324,6 +4325,7 @@ static const StreamListItem streams_list[] = {
 	create_stream_list_item<MavlinkStreamOpticalFlowRad>(),
 	create_stream_list_item<MavlinkStreamActuatorControlTarget<0> >(),
 	create_stream_list_item<MavlinkStreamActuatorControlTarget<1> >(),
+	create_stream_list_item<MavlinkStreamActuatorOutputStatus>(),
```

Comment:
```suggestion
#if defined(ACTUATOR_OUTPUT_STATUS_HPP)
	create_stream_list_item<MavlinkStreamActuatorOutputStatus>(),
#endif // ACTUATOR_OUTPUT_STATUS_HPP
```

Reply:
We've started doing this so we can easily strip down builds for smaller microcontrollers.